docs/src/createEmotionCache.ts

```diff
@@ -8,8 +8,8 @@ function globalSelector(element: Element) {
   switch (element.type) {
     case RULESET:
       element.props = (element.props as string[]).map((value: any) => {
-        if (value.match(/(:where|:is)\(/)) {
-          value = value.replace(/\.[^:]+(:where|:is)/, '$1');
+        if (value.match(/(:where|:has)\(/)) {
+          value = value.replace(/\.[^:]+(:where|:has)/, '$1');
```

Comment:
`:where|:is` is correct. They are global selectors but `:has` has to be used with an anchor.

Reply:
> Do you know where in the docs this style, which uses :is, is actually getting applied? It is for the Paper component having outlined variant. If it is not being used, we can remove it.

The initial commit: https://github.com/mui/material-ui/commit/2ca54d5671402812886f7ace56e694f5010e95c4#diff-1f70d27002ced619782b07207f4d92ac27c4918de9feaf515eb8a1728eb39c2bR358

<img width="982" alt="Screenshot 2023-05-23 at 17 21 59" src="https://github.com/mui/material-ui/assets/3165635/c265d74e-cfef-428c-a2a1-c605bbb98fe6">

Reply:
@ZeeshanTamboli I believe that it's this one at https://mui.com/templates/.
<img width="731" alt="image" src="https://github.com/mui/material-ui/assets/18292247/4b1dbd9f-17df-4a69-a805-accc3a1c3183">

Would you mind open a PR to fix it?
```diff
// brandingTheme.ts:L781
-':is(a, button)': {
+':is(a&), :is(button&)': {
  '&:hover': {
    boxShadow: `0px 4px 20px rgba(170, 180, 190, 0.3)`,
  },
},
```

Reply:
@siriwatknp It is already fixed in this PR. I just wanted to know where it's used.